docs/fleet/provision.md

```diff
@@ -164,16 +164,54 @@ This provisioning functionality is provided by the [Viam Agent provisioning subs
 
 If you specify a WiFi network to connect to in your configuration file, the Viam Agent will automatically connect to that network when in range.
```

Comment:
This is only true if it's the highest priority network, OR if roaming mode is enabled.

The "networks" section of the config should more correctly be called "additional networks" as the network info provided (via mobile app or web portal) during provisioning is the "primary network" in single network mode. Additional networks from the config are only really useful if roaming_mode is enabled.